### PR TITLE
Remove NPS survey and add more DevX tracking

### PR DESCRIPTION
**Pull request by zihejia**

### CODE REVIEW

File: Mixpanel/Controller.cs [hunk 1]
```diff
@@ -345,6 +378,9 @@ internal static void DoEngage(Value properties)
             properties["$mp_metadata"] = Metadata.GetPeopleMetadata();
 
             MixpanelStorage.EnqueueTrackingData(properties, MixpanelStorage.FlushType.PEOPLE);
+            if (Debug.isDebugBuild) {
+                MixpanelStorage.HasUsedPeople = true;
```
Comment (jaredmixpanel) [inline Mixpanel/Controller.cs:382]: no automatic people updates to check for on Unity?

File: Mixpanel/MixpanelAPI.cs [hunk 2]
```diff
@@ -257,6 +259,7 @@ public static void Track(string eventName, string key, Value value)
             Value properties = new Value();
             properties[key] = value;
             Controller.DoTrack(eventName, properties);
+            MixpanelStorage.HasTracked = true;
```
Comment (jaredmixpanel) [inline Mixpanel/MixpanelAPI.cs:262]: looks like this is set to true inside of `DoTrack()` on the line above, right?

**Review by jaredmixpanel — APPROVED** [review]
LGTM, just a couple clarifications

File: Mixpanel/Controller.cs (hunk 1 above)
Comment (zihejia) [reply Mixpanel/Controller.cs:382]: there are no automatic people updates in unity

File: Mixpanel/MixpanelAPI.cs (hunk 2 above)
Comment (zihejia) [reply Mixpanel/MixpanelAPI.cs:262]: good catch

**Pull request merged**